protobuf/kvm.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package kvm;
+
+option go_package="github.com/kubearmor/KubeArmor/protobuf";
+
+message ResponseStatus {
+    int32 status = 1;
+    string errorMessage = 2;
+}
+
+message RequestPolicyFile {
+    string agentIdentity = 1;
+    string policyLabel = 2;
+    string policyFileName = 3;
+    bytes policyFile = 4;
+}
+
+message agentIdentityInfo {
+    string agentIdentity = 1;
+}
+
+message gRPCIPAndPort {
```

Comment:
I don't think we need a message to send gRPCIPandPort... 
The KVMService will add the IP address to the KVM resource based on the GRPC connection details.